contracts/staking/StakingInfo.sol

```diff
@@ -21,9 +26,18 @@ contract IStakeManager {
     bytes32 public accountStateRoot;
     uint256 public activeAmount; // delegation amount from validator contract
     uint256 public validatorRewards;
+    // signer to Validator mapping
+    mapping(address => uint256) public signerToValidator;
```

Comment:
Shouldn't this be a function?  Why need to use storage for the same?

Reply:
this is just a dummy interface, but yes one function will also do. Thanks